src/utils/common.ts

```diff
@@ -173,3 +173,18 @@ export const scrollOnPageChange = (containerId: string) => {
         scrollToHash(containerId);
     }
 };
+
+export const getCommonQa = (qa?: string, customKeys: Array<string> = []) => {
+    const qaObject: Record<string, string> = {};
+
+    if (qa) {
+        const commonKeys = ['container', 'content', 'wrapper', 'image', 'button'];
```

Comment:
everytime when func will be called, `commonKeys` will be created, maybe we need to set this constant in module level ?

Reply:
sure